test/e2e/framework/util.go

```diff
@@ -123,7 +123,7 @@ const (
 	NodeReadyInitialTimeout = 20 * time.Second
 
 	// How long pods have to be "ready" when a test begins.
-	PodReadyBeforeTimeout = 2 * time.Minute
+	PodReadyBeforeTimeout = 3 * time.Minute
```

Comment:
If 2 minutes is not enough, we should probably make it higher than 3 minutes...
Maybe 5 minutes?